src/pagination/pagination.styles.tsx

```diff
@@ -131,9 +130,28 @@ export const PageItem = styled(Button.Default)<StyleProps>`
     :hover,
     :active,
     :focus {
-        color: ${(props) =>
-            props.$selected ? Color.Neutral[8] : Color.Primary};
         box-shadow: none;
```

Comment:
I missed this previously as well, but there's a separate style for `:active` https://www.figma.com/file/us90jOpWBahsqK2VAluPvD/Flagship-Design-System?type=design&node-id=3648%3A37199&t=PormYZVzdmBKNQTp-1